addon/templates/components/bs-form/element.hbs

```diff
@@ -1 +1,65 @@
-{{partial formElementTemplate}}
\ No newline at end of file
+{{#component
```

Comment:
Just for formatting, can you bring `layoutComponent` up onto the same line to differentiate the variable with the component being invoked from all the arguments being passed to it?